### PR TITLE
Add Op (col2im) | feat(atenlib)

### PR DESCRIPTION
**Pull request by xiaowuhu**

### CODE REVIEW

File: onnxscript/function_libs/torch_aten/ops/nn.py [hunk 1]
```diff
@@ -213,17 +213,25 @@ def aten_celu(self: FLOAT, alpha: float = 1.0) -> FLOAT:
...
+    self: TReal,
     output_size: INT64,
     kernel_size: Sequence[int],
-    dilation: Sequence[int],
-    padding: Sequence[int],
-    stride: Sequence[int],
-) -> TensorType:
+    dilation: Sequence[int] = [1, 1],
+    padding: Sequence[int] = [0, 0, 0, 0],
+    stride: Sequence[int] = [1, 1],
```
Comment (titaiwangms) [inline onnxscript/function_libs/torch_aten/ops/nn.py:223]: I don't find these three having default value on PyTorch. Am I missing it?

Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/nn.py:223]: I agree. We don't need to fill in defaults when the aten op does not specify it

File: onnxscript/function_libs/torch_aten/ops/nn.py [hunk 2]
```diff
@@ -225,13 +225,48 @@ def aten_col2im(
...
+
+    # The strides should be [x, y]
+    if isinstance(stride, int):  # x -> [x, x]
+        strides = [stride] * expand_size
+    elif stride is None:
+        strides = kernel_shape
+    else:
+        strides = stride
+
+    # Only one ONNX op here so didn't write an private function
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/nn.py]: SG

File: onnxscript/tests/function_libs/torch_aten/extra_opinfo.py [hunk 3]
```diff
@@ -215,7 +216,36 @@ def sample_inputs_max_pool2d_with_indices(
...
+    )
+
+    make_arg = functools.partial(torch_testing.make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    for shape, output_size, kernel_size, dilation, padding, stride in cases:
+        tensor = make_arg(shape)
+        yield opinfo_core.SampleInput(tensor, output_size, kernel_size, dilation, padding, stride)
+
+
 OP_DB: List[opinfo_core.OpInfo] = [
+    opinfo_core.OpInfo('nn.functional.fold',  # This name must be a valid function in torch
```
Comment (justinchuby) [inline onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: Is this right? I am pretty sure the name can be anything?

File: onnxscript/function_libs/torch_aten/ops/nn.py [hunk 4]
```diff
@@ -225,13 +225,48 @@ def aten_col2im(
 ) -> TReal:
     """col2im(Tensor self, SymInt[2] output_size, int[2] kernel_size, int[2] dilation, int[2] padding, int[2] stride) -> Tensor"""
 
+    # Torch prefer to use single number x for kerne,stride,pad,dilation on both side implicitly
+    # But ONNX needs pair number [x,y] to specify on each side explicitly
+    expand_size = 2
+
+    # The dilations should be [x, y]
+    if isinstance(dilation, int):  # x -> [x, x]
```
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_aten/ops/nn.py:232]: Should we put dilation,padding,stride into kwargs? Because op.Col2im() need them as attribute.

File: onnxscript/tests/function_libs/torch_aten/extra_opinfo.py (hunk 3 above)
Comment (xiaowuhu) [reply onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: If change to nn.functional.col2im will get a failed.

Comment (xiaowuhu) [reply onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: error

File: onnxscript/function_libs/torch_aten/ops/nn.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_aten/ops/nn.py:232]: Change dilation,padding,stride to attribute. Because if the dilation = (1,1), it will be convert to tensor([1,1]), then op.Col2im(... dilations=dilation...) will failed in FullGraph mode.

File: onnxscript/tests/function_libs/torch_aten/extra_opinfo.py (hunk 3 above)
Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: Let me try something

Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: I just renamed to col2im. I will see if it works and will revert if it doesn't

**Review by justinchuby — APPROVED** [review]

File: onnxscript/tests/function_libs/torch_aten/extra_opinfo.py (hunk 3 above)
Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: Fixed

Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/extra_opinfo.py]: The trick is to populate the `op` field of `OpInfo`
